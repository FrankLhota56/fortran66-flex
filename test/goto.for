C
C GO TO / Label Test 
C
090   ASSIGN 100 TO I2
      Ass ign 99  To  i 1
     0Assign 666
     1To I3
      Go to 666
      GOTO I1, (99, 100)
100   GO
     XTO (99, 100), I2
      END
