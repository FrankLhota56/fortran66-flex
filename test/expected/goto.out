Line     4: INT_LIT(90)
Line     4: ASSIGN
Line     4: INT_LIT(100)
Line     4: TO
Line     4: ID(i2)
Line     5: EOS
Line     5: ASSIGN
Line     5: INT_LIT(99)
Line     5: TO
Line     5: ID(i1)
Line     6: EOS
Line     6: ASSIGN
Line     6: INT_LIT(666)
Line     7: TO
Line     7: ID(i3)
Line     8: EOS
Line     8: GO
Line     8: TO
Line     8: INT_LIT(666)
Line     9: EOS
Line     9: GO
Line     9: TO
Line     9: ID(i1)
Line     9: ','
Line     9: '('
Line     9: INT_LIT(99)
Line     9: ','
Line     9: INT_LIT(100)
Line     9: ')'
Line    10: EOS
Line    10: INT_LIT(100)
Line    10: GO
Line    11: TO
Line    11: '('
Line    11: INT_LIT(99)
Line    11: ','
Line    11: INT_LIT(100)
Line    11: ')'
Line    11: ','
Line    11: ID(i2)
Line    12: EOS
Line    12: END
